concept string methods minor arrangement (#2329)
exercism/python: exercises/concept/little-sisters-essay/.docs/instructions.md

> In this exercise you are helping your younger sister edit her paper for school. The teacher is looking for correct punctuation, grammar, and excellent choice of words.

Reviewer: ```suggestion
In this exercise you are helping your younger sister edit her paper for school. The teacher is looking for correct punctuation, grammar, and excellent word choice.
```

> In this exercise, you are helping your younger sister edit her paper for school. The teacher is looking for correct punctuation, grammar, and excellent choice of words.

Author: Wow, true, but weird. Thanks

## 3. Clean up spacing

> To make the paper _even better_, you can replace some of the adjectives with synonyms. Write the function `replace_word_choice()` that takes `sentence`, `old_word`, and `new_word` as parameters.

Reviewer: ```suggestion
To make the paper look professional, unnecessary spacing needs to be removed. Implement the function `remove_extra_spaces()` which `sentence` as a parameter, and removes extra whitespace at the beginning and at the end of the sentence. This function should return a `str`.
```

> This function should replace all instances of the `old_word` with the `new_word`, and return a new `str` with the updated sentence.

Reviewer: ```suggestion
```

Reviewer: I think this was a mis-cut and paste.

## 3. Clean up spacing

To make the paper look professional, unnecessary spacing needs to be removed. Implement the function `remove_extra_spaces()` which `sentence` as a parameter, and removes extra whitespace at the beginning and at the end of the sentence. 
This function should return a `str`.

> This function should replace all instances of the `old_word` with the `new_word`, and return a new `str` with the updated sentence.

Reviewer: This sentence needs to be removed....it belongs to the task below.

> This function should replace all instances of the `old_word` with the `new_word`, and return a new `str` with the updated sentence.

Reviewer: ```suggestion
```

Reviewer: This sentence needs to be removed....it belongs to the task below.
